src/mozanalysis/utils.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import pyspark.sql.functions as F
+from pyspark.sql.column import Column
+
+
+def dedupe_columns(columns):
+    """
+    Given a list of columns, returns a list with duplicates removed.
+
+    These can be either from `F.col` or `F.expr`.
```

Comment:
I think a longer explanation of why we need to do this instead of using `set` would be useful, including the context with test failing on python 3, in the off chance the underlying assumptions change in the future